lib/LocalVerifiableCredentialStore.js

```diff
@@ -229,10 +230,15 @@ export class LocalVerifiableCredentialStore {
         }
       }
       const queryType = query.map(q => q.type);
+      const requiredIssuer = query.map(q => q.issuer);
```

Comment:
I don't really understand why `query` is getting built inside the for loop above only to split it back out again here. We started with `types`, which we iterate over and add to `query` -- and then we run a map function to just get the types back out again and store them in `queryTypes`. Doesn't that not really accomplish anything?

We do the same thing with issuers... except now we may get `undefined` issuers when `issuers.length === 0`, right?

I may be missing something, but it seems like this code needs to be cleaned up.

Reply:
I've done some clean up here 672959e.